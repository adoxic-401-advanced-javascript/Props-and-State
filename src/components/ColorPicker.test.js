import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('Color Picker component', () => {
  it('renders Color Picker', () => {
    const wrapper = shallow(<ColorPicker selectColor={() => {}}
      colors={['blue']}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
