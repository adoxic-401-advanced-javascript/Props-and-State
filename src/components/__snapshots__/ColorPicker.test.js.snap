// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color Picker component renders Color Picker 1`] = `
<Fragment>
  <button
    key="blue"
    onClick={[Function]}
  >
    blue
  </button>
</Fragment>
`;
